cmd/export-distro/main.go

```diff
@@ -11,67 +11,70 @@ package main
 import (
 	"flag"
 	"fmt"
+	"net/http"
 	"os"
 	"os/signal"
+	"strconv"
 	"syscall"
+	"time"
 
 	"github.com/edgexfoundry/edgex-go"
 	"github.com/edgexfoundry/edgex-go/internal"
 	"github.com/edgexfoundry/edgex-go/internal/export/distro"
-	"github.com/edgexfoundry/edgex-go/internal/pkg/config"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/startup"
 	"github.com/edgexfoundry/edgex-go/internal/pkg/usage"
 	"github.com/edgexfoundry/edgex-go/pkg/clients/logging"
-	"github.com/edgexfoundry/edgex-go/pkg/models"
 )
 
+var bootTimeout = 30000 //Once we start the V2 configuration rework, this will be config driven
+
 func main() {
-	var (
-		useConsul  bool
-		useProfile string
-	)
+	start := time.Now()
+	var useConsul bool
+	var useProfile string
+	
 	flag.BoolVar(&useConsul, "consul", false, "Indicates the service should use consul.")
 	flag.BoolVar(&useConsul, "c", false, "Indicates the service should use consul.")
 	flag.StringVar(&useProfile, "profile", "", "Specify a profile other than default.")
 	flag.StringVar(&useProfile, "p", "", "Specify a profile other than default.")
 	flag.Usage = usage.HelpCallback
 	flag.Parse()
 
-	configuration := &distro.ConfigurationStruct{}
-	err := config.LoadFromFile(useProfile, configuration)
-	if err != nil {
-		logBeforeInit(fmt.Errorf("%s: version: %s: err: %s", internal.ExportDistroServiceKey, edgex.Version, err.Error()))
+	params := startup.BootParams{UseConsul: useConsul, UseProfile: useProfile, BootTimeout: bootTimeout}
+	startup.Bootstrap(params, distro.Retry, logBeforeInit)
+
+	ok := distro.Init()
```

Comment:
ove definition into `if`, like `if ok :=...; !ok {...`

Reply:
`if ok := distro.Init()` doesn't work because the assignment function doesn't return a value.